src/main/java/study/carracing/domain/Cars.java

```diff
@@ -0,0 +1,26 @@
+package study.carracing.domain;
+
+import study.carracing.ui.ResultView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private List<Car> carList = new ArrayList<>();
```

Comment:
```suggestion
    private List<Car> items = new ArrayList<>();
```

`List`, `Map`, `Set` 등 컬렉션의 이름은 사용하지 않는걸 추천드립니다. ([참고자료](https://tecoble.techcourse.co.kr/post/2020-04-24-variable_naming))

---

지금 작성하신 방법처럼 초기화한 다음 매번 add를 하는 방식으로 사용할 수도 있겠으나
이미 만들어진 데이터를 대입하여 사용할 수 있다면 그렇게 만드는 편이 깔끔하다고 생각합니다.
내부 데이터를 불변으로 만들 수 있기 때문입니다.

무조건 불변일 필요는 없습니다만, 일급 컬렉션이 줄 수 있는 장점 중 하나라고 생각하여 의견 달아봅니다.

Reply:
자동차들이라는 의미를 변수명에 담고 싶어 `cars`는 유지하되 
클래스명을 `RacingCars`로 변경하였습니다.